PySDM/backends/impl_numba/methods/displacement_methods.py

```diff
@@ -44,14 +47,17 @@ def calculate_displacement_body_1d(
                 displacement,
                 courant,
                 position_in_cell,
+                cell_id,
                 n_substeps,
+                enable_monte_carlo,
+                rng,
```

Comment:
Here it would be something like this:

```suggestion
                rng[droplet],
```

The same for `2d` and `3d`